.travis.yml

```diff
@@ -80,6 +80,20 @@ matrix:
         - npm run build -- --scope "@loopback/test-repository-postgresql" --include-dependencies
         - psql -U postgres -a -f acceptance/repository-postgresql/test/schema.sql
         - cd acceptance/repository-postgresql && npm run mocha
+    - node_js: '10'
+      os: linux
+      env:
+        - TASK=test-repository-cloudant
+      install:
+        # - rm -rf node_modules && yarn cache clean
+        - npm ci && npm run docker:setup && source cloudant-config.sh && pwd
+
+      script:
+        - source cloudant-config.sh
+        - npm run postinstall -- --scope "@loopback/test-repository-cloudant" --include-dependencies
+        - npm run postinstall -- --include-dependencies
+        - npm run build #-- --scope "@loopback/test-repository-cloudant" --include-dependencies
+        - cd acceptance/repository-cloudant && npm run mocha
```

Comment:
This is where you should source your config.. `source cloudant-config.sh && cd acceptance/repository-cloudant && npm run mocha`